mifospay/src/main/java/org/mifos/mobilewallet/mifospay/savedcards/ui/CardsScreen.kt

```diff
@@ -0,0 +1,292 @@
+package org.mifos.mobilewallet.mifospay.savedcards.ui
+
+
+import android.widget.Toast
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material.DropdownMenuItem
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Add
+import androidx.compose.material.icons.filled.Close
+import androidx.compose.material.icons.filled.Search
+import androidx.compose.material3.AssistChip
+import androidx.compose.material3.AssistChipDefaults
+import androidx.compose.material3.Card
+import androidx.compose.material3.DropdownMenu
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.SearchBar
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
+import com.mifos.mobilewallet.model.entity.savedcards.Card
+import org.mifos.mobilewallet.mifospay.MifosPayApp
+import org.mifos.mobilewallet.mifospay.designsystem.component.MifosOverlayLoadingWheel
+import org.mifos.mobilewallet.mifospay.designsystem.theme.MifosTheme
+import org.mifos.mobilewallet.mifospay.savedcards.presenter.CardsUiState
+import org.mifos.mobilewallet.mifospay.utils.Constants
+import java.util.Locale
+
```

Comment:
Create here another function where you will pass viewModel and onAddBtn:() -> Unit 
`fun CardsScreen(
viewModel: CardsScreenViewModel = hiltViewModel(), 
  onAddBtn:() -> Unit
  )`
, and inside that function you will call below CardScreen function. 

using this you can easily generate the preview by just passing the sample values in preview function.